src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1168,7 +1168,13 @@ def get_eventual_alias(self) -> Optional[AliasInfo]:
         tbl_expression = self.get_child("table_expression")
         if not tbl_expression:  # pragma: no cover
             tbl_expression = self.get_child("bracketed").get_child("table_expression")
-        ref = tbl_expression.get_child("object_reference")
+        # For TSQL nested, bracketed tables get the first table as reference
+        if tbl_expression and not tbl_expression.get_child("object_reference"):
+            if tbl_expression.get_child("bracketed"):
+                tbl_expression = tbl_expression.get_child("bracketed").get_child(
```

Comment:
Nitpick: I think this will work if there's _at most_ one level of parentheses. Should we worry about handling two or more levels?

Reply:
I’m ok leaving this as is for now. It’s similar to the check above with similar restrictions. Also once you start allowing multiple levels you could be in expressions and sub clauses and all sorts. So I say leave for now.

Reply:
I agree. Just wanted to make it a conscious decision. Might be worth adding a comment noting this, but really no big deal.

Reply:
Let’s leave for now. Not even sure the syntax is valid with multiple delimiters and comment might suggest there definitely is a valid syntax like this that is not supported.